src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/GlobalSearchResource.kt

```diff
@@ -1,37 +1,28 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.resource
 
-import io.swagger.annotations.ApiImplicitParam
-import io.swagger.annotations.ApiImplicitParams
-import io.swagger.annotations.ApiOperation
-import io.swagger.annotations.ApiParam
-import org.springframework.data.domain.Page
+import io.swagger.v3.oas.annotations.Operation
 import org.springframework.data.domain.Pageable
 import org.springframework.data.web.PageableDefault
 import org.springframework.http.MediaType
 import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RestController
-import uk.gov.justice.digital.hmpps.prisonersearch.model.Prisoner
 import uk.gov.justice.digital.hmpps.prisonersearch.services.GlobalSearchCriteria
 import uk.gov.justice.digital.hmpps.prisonersearch.services.GlobalSearchService
 
 @RestController
 @Validated
-class GlobalSearchResource(
-  val globalSearchService: GlobalSearchService
-) {
+class GlobalSearchResource(private val globalSearchService: GlobalSearchService) {
 
-  @PostMapping("/global-search", produces = [MediaType.APPLICATION_JSON_VALUE], consumes = [MediaType.APPLICATION_JSON_VALUE])
-  @ApiOperation(value = "Match prisoners by criteria", notes = "Requires GLOBAL_SEARCH role")
-  @ApiImplicitParams(
```

Comment:
these aren't implicit since there is a `Pageable` that is passed through, docs look to have been generated fine